dbaas/notification/tasks.py

```diff
@@ -653,6 +654,39 @@ def check_ssl_expire_at(self):
         return
 
 
+@app.task(bind=True)
+@only_one(key="changemongodblogrotate")
+def change_mongodb_log_rotate(self):
+    LOG.info("Retrieving all restarted MongoDB databases")
+    today = date.today()
+    worker_name = get_worker_name()
+    task = TaskHistory.register(
+        request=self.request, user=None, worker_name=worker_name)
+    task.relevance = TaskHistory.RELEVANCE_CRITICAL
+    one_month_later = today + timedelta(days=30)
+    mongodb_rotate_envs = Configuration.get_by_name('mongodb_rotate_envs')
+    extra_filters = {}
+    if mongodb_rotate_envs:
+        extra_filters = {'databaseinfra__environment__name__in': mongodb_rotate_envs.split(',')}
+    try:
+        instances = Instance.objects.filter(
+            instance_type__in=[Instance.MONGODB, Instance.MONGODB_ARBITER]
+        )
+        mongodb_restarted_instances = []
+        restart_database_managers = RestartDatabase.objects.all()
```

Comment:
I think is better u use TaskSchedule object. Using that the instances will be repeated